src/main/java/com/openlattice/datastore/data/controllers/DataIntegrationController.java

```diff
@@ -45,8 +46,8 @@
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
-import static com.openlattice.authorization.EdmAuthorizationHelper.WRITE_PERMISSION;
-import static com.openlattice.authorization.EdmAuthorizationHelper.aclKeysForAccessCheck;
+import static com.openlattice.authorization.EdmAuthorizationHelper.*;
```

Comment:
🤔

Reply:
my autoformatter does this

Reply:
Oh, I just made this face b/c `READ_PERMISSION` is being imported individually despite the `.*` import

Reply:
oh wut. it's gone now.